docs/admin/hardware.rst

```diff
@@ -0,0 +1,83 @@
+
+Recommended hardware
+====================
+
+.. include:: ../includes/top-warning.rst
+
+Qubes OS hardware requirements
+------------------------------
+
+In order to install and use SecureDrop Workstation, you will need a Qubes-Compatible computer with the following speifications:
+
+- 64-bit Intel or AMD processor with virtualization support
+- a minimum of 16GB RAM (32GB recommended for production use)
+- sufficient disk space for the Qubes OS base install and SecureDrop Workstation VMs (a 128GB or greater SSD is recommended)
+
+More information on hardware compatibility can be found on the `Qubes OS System Requirements <https://www.qubes-os.org/doc/system-requirements/>`_ page, and information on specific systems can be found via the `hardware compatibility list <https://www.qubes-os.org/hcl/>`_.
+
+In order to print submissions, a supported non-networked printer is required. Supported models currently include:
+
+- Brother HL-L2320D 
+- HP LaserJet Pro M404n 
+
+More printer options will be added in future releases.
+
+Lenovo ThinkPad T480
+--------------------
+The ThinkPad T480 is a recommended option for SecureDrop Workstation, as it is being used by the core team for development and testing. If you plan to use it, you should follow the instructions below to ensure that the BIOS is up to date before proceeding with the installation:
+
+.. _t480_bios:
+
+Upgrading the T480 BIOS
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The instructions below assume the use of a Linux-based computer for the creation of a BIOS upgrade USB. To upgrade the T480 BIOS:
+
+- Locate the machine type of the T480 - it be found via the ``Main`` tab in Thinkpad Setup (accessed by pressing **Enter** on startup). For recent T480s, it will start with a string like `20L5` or `20L6`.
```

Comment:
To avoid confusion, I would say something like "it will _be_ a string", not "start with a string", since the full machine type number is _not_ what they want (it won't produce matches on the support portal)

Reply:
:heavy_check_mark: